include/CommandLineSetup.h

```diff
@@ -16,6 +16,9 @@ class CommandLineParameters {
     bool serialMode = true;    // -s FLAG (NO ARGUMENT) DEFAULT=TRUE OPTIONAL
     int nThreads = 0;          // -t Number of threads, only active if serialMode is set to false OPTIONAL
 
+    int argc{};
+    char** argv = nullptr;
```

Comment:
Why this `args` are needed?

Reply:
https://github.com/rest-for-physics/restG4/pull/41/files#r896892579

Reply:
Well, you don't have to call them `argc` and `argv` I think is misleading.

Reply:
> Well, you don't have to call them `argc` and `argv` I think is misleading.

Well they hold the same data, how would you call them then? feel free to make a change.

Reply:
I would suggest `cmdArgc` and `cmdArgv`, I will provide a commit.

Btw, I just figure out that these `args` are only used here:
```
#ifdef G4UI_USE
        auto ui = new G4UIExecutive(commandLineParameters.argc, commandLineParameters.argv);
```
Is `G4UI_USE` still supported? Otherwise I think we can completely remove the `args` stored on `CommandLineSetup.h`

Reply:
> I would suggest `cmdArgc` and `cmdArgv`, I will provide a commit.
> 
> Btw, I just figure out that these `args` are only used here:
> 
> ```
> #ifdef G4UI_USE
>         auto ui = new G4UIExecutive(commandLineParameters.argc, commandLineParameters.argv);
> ```
> 
> Is `G4UI_USE` still supported? Otherwise I think we can completely remove the `args` stored on `CommandLineSetup.h`

We support Geant4 v10.4.3 so I guess we should still maintain this.